pkgs/development/python-modules/b2sdk/default.nix

```
@@ -0,0 +1,24 @@
+{ stdenv, buildPythonPackage, fetchPypi, setuptools_scm
+, requests, arrow, logfury, tqdm }:
+
+buildPythonPackage rec {
+  pname = "b2sdk";
+  version = "1.1.4";
```

```suggestion
  version = "1.1.4";
  disabled = isPy27;
```

really old version of arrow
```
  ERROR: Could not find a version that satisfies the requirement arrow<0.13.1,>=0.8.0; python_version <= "3.4" (from b2sdk==1.1.4) (from versions: none)
  ERROR: No matching distribution found for arrow<0.13.1,>=0.8.0; python_version <= "3.4" (from b2sdk==1.1.4)
```